index.js

```diff
@@ -115,7 +115,7 @@ async function exportPodcasts(podcastsDBData) {
     filesWithDBData.map(async (podcast) => {
       // Create an export subdir
       let exportDirPath = outputDir;
-      const exportDir = podcast.dbMeta?.zpodcast;
+      const exportDir = podcast.dbMeta?.zpodcast.replaceAll('/', '_');
```

Comment:
I'm afraid that there are could be more symbols, I think `sanitize-filename` (here just `sanitize`) can be used? maybe you can try it for your case?

Reply:
Thanks, works great.  I'm going to close this PR as the slash-fix is incorporated in #6 .  Cheers!